nautobot/core/templates/responsive_table.html

```diff
@@ -1,5 +1,5 @@
 {% load render_table from django_tables2 %}
 
 <div class="table-responsive">
-    {% render_table table table_inc_template|default:'inc/table.html' %}
+    {% include table_inc_template|default:'inc/table.html' with list_view=True %}
```

Comment:
After manual testing, this is functionally the same as `render_table table`

Reply:
Looking at the `render_table` definition, the main concern I see is with regard to how it handles TemplateColumn: (https://github.com/jieter/django-tables2/blob/master/django_tables2/templatetags/django_tables2.py#L158-L164)

Reply:
Would it be acceptable to use a separate template for list view table then? It is difficult to pass in a django template variable with the `with` keyword in this line `{% render_table table table_inc_template|default:'inc/table.html' %}.`

Reply:
I'm not sure how important that bit of code is TBH - `blame` shows it as having been untouched in the last 12 (!) years. :-) If you've tested with some various list views and the data all look OK then I'm willing to have us take that risk, personally.

Reply:
Otherwise yeah creating a new template file might be appropriate.

Reply:
Actually, there's already `nautobot/core/templates/panel_table.html`... maybe we can/should be using it in the list view?

Reply:
Wow didnt see that at all, Im going to try using that.

Reply:
The only noticable difference using`panel_table.html` is that when a table has no entries. It is rendered as such:
![Screen Shot 2023-11-13 at 1 52 37 PM](https://github.com/nautobot/nautobot/assets/46973263/33e01fdd-1269-43f1-a759-d1b80c524d59)

Reply:
I'm not a huge fan of that but I can accept it. We have so many fiddly variants of a basic object table that I'd rather we not add another new one. :-(